### PR TITLE
New connect no sql branch

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Same data but another branch coz I'm working from new machine

### CODE REVIEW

**Comment by DaniilPoiarkov** [comment]
Screenshot_1
pm.tests for this build/10 iterations

File: backend/Tasque.BLL/Services/TaskService.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            {
+                Id = task.Id.ToString(),
+                CustomFields = model?.CustomFields,
+            };
+
+            var customAttributes = await _cosmosTaskService.UpdateTask(cosmosModel);
+
+            task.Description = model.Description;
+            task.Summary = model.Summary;
+            //task.Labels = model.Labels;
```
Comment (ismv) [inline backend/Tasque.BLL/Services/TaskService.cs]: will we need these commented lines?

File: backend/Tasque.Core.Common/DTO/AttachmentDto.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.Common.DTO
+{
+    public class AttachmentDto
+    {
+        public string URI { get; set; }
```
Comment (ismv) [inline backend/Tasque.Core.Common/DTO/AttachmentDto.cs]: github actions warnings

File: backend/Tasque.BLL/Services/TaskService.cs (hunk 1 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.BLL/Services/TaskService.cs]: Done

File: backend/Tasque.Core.Common/DTO/AttachmentDto.cs (hunk 2 above)
Comment (DaniilPoiarkov) [reply backend/Tasque.Core.Common/DTO/AttachmentDto.cs]: Done

**Comment by DaniilPoiarkov** [comment]
Postman TemplateTests, commit: 9f838d5
pm tests

**Comment by DaniilPoiarkov** [comment]
TaskService pm tests:
image
TemplateService pm tests:
image

Commit: a9a9791

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by andpii — APPROVED** [review]

**Pull request merged**